modules/order/tests/src/Functional/OrderRefreshSettingsTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\Tests\commerce_order\Functional;
+
+use Drupal\Tests\commerce\Functional\CommerceBrowserTestBase;
+use Drupal\commerce_order\Entity\OrderType;
+
+/**
+ * Tests the cart refresh.
```

Comment:
Because direct refactor of directory and namespace from cart module. I'm going to just merge them